eessi/testsuite/eessi_mixin.py

```diff
@@ -113,6 +115,16 @@ def measure_mem_usage(self):
             # instead of the @performance_function decorator
             self.perf_variables['memory'] = make_performance_function(hooks.extract_memory_usage, 'MiB', self)
 
+    @run_after('init', always_last=True)
+    def set_tag_ci(self):
+        "Set CI tag if bench_name_ci and bench_name are set and are equal"
+        if self.bench_name_ci:
+            if not self.bench_name:
+                msg = "Attribute bench_name_ci is set, but bench_name is not set"
+                raise ReframeFatalError(msg)
+            if self.bench_name == self.bench_name_ci:
+                self.tags.add(TAGS['CI'])
```

Comment:
Assuming we _do_ keep this: should we log this as well (as was done before in the cp2k class)

Reply:
addressed in https://github.com/EESSI/test-suite/pull/204